### DISCUSSION TITLE
LIDARR - Clearing Previously downloaded?

### DISCUSSION THREAD

**Discussion by Icenova20** [RandomNinjaAtk/arr-scripts#80] [Q&A] [answered]
I think somewhere along the line my audio script and lidar database got out of whack, I'll notice in the audio script that albums will be marked as previously downloaded but the files don't exist in my file system. 

I think this may have something to do with files that already existed when first starting to use the script, but starting with a fresh lidar database. Although I cant be certain. Could also be that the way i was running docker there wasnt enough compute to properly import the files and something happened there.

I suppose my main question would be, how do i clear what has been previously downloaded?

What would be the repercussions of that? would every file attempt to download again? or only those marked missing?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Deleting the /config/extended folder will effectively reset everything.

You delete it and restart the container.


The scripts only look for what lidarr tells it to, so it shouldn’t be that much of an issue. You’ll end up redownloading things that the script matched but lidarr couldn’t import due to not being able to match it, but that  should be a small percentage of things…
